src/jit/gentree.h

```diff
@@ -1307,7 +1307,7 @@ struct GenTree
         return OperIsShiftOrRotate(OperGet());
     }
 
-    bool OperIsArithmetic() const
+    bool OperIsArithmetic() const // TODO-Throughput: This seems expensive, and is frequently called.
```

Comment:
Currently this is not a problem, the checked values form a continuous range so the generated code looks something like this (in VC++):
```asm
movzx       eax,byte ptr [rdi+8]  
lea         ecx,[rax-2Eh]  
cmp         cl,0Eh  
ja          Compiler::fgMorphForRegisterFP+0E7h (07FFAF52E3137h)  
```
In general, from what I've seen at least VC++ is pretty good at handling such checks.